### ISSUE TITLE
Left Sidebar is Bugged for Me

### ISSUE THREAD

**Issue by Tyrichyrich** [Godiesc/opera-gx#19]
Hi, I'm back again and I found one of your extras pretty nice, sidebar, but it seems to not work. It just leaves a blank sidebar, unless when you put the mouse over it... I'll send an image of the problem if I can, thx for adding the sidebar though.

**Comment by Tyrichyrich** [comment]
![Screenshot (9)](https://user-images.githubusercontent.com/110419964/194703974-2b89cf0f-9557-4f47-89d8-17a65a08a05e.png)

**Comment by Tyrichyrich** [comment]
![Screenshot (10)](https://user-images.githubusercontent.com/110419964/194703996-0803413b-349d-4a17-9b3d-0196e72ea59d.png)

**Comment by Godiesc** [comment]
hi, are you using a compatible theme? I can´t see in your screenshot the complete tab line either.

**Comment by Tyrichyrich** [comment]
Yes, I realized what I did wrong I wasn't using extension now I think I have it working now.

**Comment by Tyrichyrich** [comment]
So, my "fix" and redownloading didn't help... What do I do now?

**Comment by Godiesc** [comment]
hi, give me information of your OS, version of firefox, and how you download the theme?

**Comment by Tyrichyrich** [comment]
I downloaded the theme today and the CSS today. My os is windows 11 pre- 22h2 and version is 105.0.3 or the newest version.

**Comment by Tyrichyrich** [comment]
The main problem is that the firefox toolbar bookmark things disapear, but regular bookmarks, expect for the ones that get an automaticaly added website logo, and extensions.

**Comment by Godiesc** [comment]
your main language is english? I speak spanish and is easy for me to chat whit it, but if you want in english is good too.
And first of all if you are using a pre-release od your OS maybe that could affect it. The theme dont shows any issues in windows 10 (I use it ) and the compatibility to windows 11 is strong.
 -- ¿when you install the base theme, without any "extra" the gradient line is show?

**Comment by Tyrichyrich** [comment]
Not a pre release just a version before 22h2 should have said it better. Yes, my main language is English.

**Comment by Godiesc** [comment]
well,  maybe the theme isn't compatible with windows11 anymore if the base theme don't work for you.

**Comment by Tyrichyrich** [comment]
No the base css works it's just the extra's sidebar, I think the color theme works fine though. Are you talking about the firefox "theme" or the css? Because if it's the color I'll change it to see if that'll fix or reinstall it.

**Comment by Godiesc** [comment]
I mean [this themes](https://github.com/Godiesc/opera-gx#temas---themes), and if the base theme (without extras) works fine maybe you add extra code or import other css files? do you tested only my theme?
 I could help you with "quick assistance" program (is a windows default app) if you want, check the program and let me know if you can.

**Comment by Tyrichyrich** [comment]
I only used your css and the red color theme, the first one that starts with an f. Should I reinstall anything?

**Comment by Godiesc** [comment]
I don't use windows 11 so it's difficult to me test what would be the problem but like you tell me the only problem is the specific css to the left sidebar, you have knowledge to edit css?

**Comment by Tyrichyrich** [comment]
I have no knowledge sadly, I'll just keep this open for now, but Windows 10 and 11 are similar, so I can't think it's that.

**Comment by Tyrichyrich** [comment]
I decided that I don't really "need" it like I thought. Thank you very much though with trying to being a good dev and trying to reach out to the community!😁

**Comment by Godiesc** [comment]
sorry if the theme don't work for you, A custom windows theme (that theme that change a lot of the UI of the OS) always affect negatively that kind of themes, maybe you have one installed but if not I can't imagine the problem.
❣️

**Comment by Tyrichyrich** [comment]
Hey, I'm still using it, just not the sidebar...

**Comment by Godiesc** [comment]
hi, I recognize the problem now, and its caused by a rule in the user.js, maybe you didn't copy that file in your profile folder.
the specific config  "layout.css.color-mix.enabled" should be set "true" in about:config
